sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/TextAnalyticsRestClient.cs

```diff
@@ -933,5 +933,150 @@ public Response<SentimentResponse> Sentiment(MultiLanguageBatchInput input, stri
+            }
+        }
+
+        /// <summary> Get details of the healthcare prediction job specified by the jobId. </summary>
+        /// <param name="nextLink"> Next link for list operation. </param>
+        /// <param name="showStats"> (Optional) if set to true, response will contain request and document level statistics. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="nextLink"/> is null. </exception>
+        public Response<HealthcareJobState> HealthStatusNextPage(string nextLink, bool? showStats = null, CancellationToken cancellationToken = default)
+        {
+            if (nextLink == null)
+            {
+                throw new ArgumentNullException(nameof(nextLink));
+            }
+
+            using var message = CreateHealthStatusNextPageRequest(nextLink, showStats);
+            _pipeline.Send(message, cancellationToken);
+            switch (message.Response.Status)
+            {
+                case 200:
+                    {
+                        HealthcareJobState value = default;
+                        using var document = JsonDocument.Parse(message.Response.ContentStream);
+                        value = HealthcareJobState.DeserializeHealthcareJobState(document.RootElement);
+                        return Response.FromValue(value, message.Response);
+                    }
+                default:
+                    throw _clientDiagnostics.CreateRequestFailedException(message.Response);
+            }
+        }
+
+        internal HttpMessage CreateHealthStatusNextPageNextPageRequest(string nextLink, bool? showStats)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendRaw("/text/analytics/v3.1-preview.4", false);
+            uri.AppendRawNextLink(nextLink, false);
+            request.Uri = uri;
+            request.Headers.Add("Accept", "application/json, text/json");
+            return message;
+        }
+
+        /// <summary> Get details of the healthcare prediction job specified by the jobId. </summary>
+        /// <param name="nextLink"> The URL to the next page of results. </param>
+        /// <param name="showStats"> (Optional) if set to true, response will contain request and document level statistics. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="nextLink"/> is null. </exception>
+        public async Task<Response<HealthcareJobState>> HealthStatusNextPageNextPageAsync(string nextLink, bool? showStats = null, CancellationToken cancellationToken = default)
```

Comment:
this is created twice and it has already been reported: https://github.com/Azure/autorest.csharp/issues/1056
I made the name very very ugly so no one uses it :P